js/modules/k6/metrics/metrics.go

```diff
@@ -65,7 +65,18 @@ func newMetric(ctxPtr *context.Context, name string, t stats.MetricType, isTime
 	}
 
 	rt := common.GetRuntime(*ctxPtr)
-	return common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr), nil
+	binded := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
```

Comment:
```suggestion
	bound := common.Bind(rt, Metric{stats.New(name, t, valueType)}, ctxPtr)
```
https://www.oxfordlearnersdictionaries.com/definition/american_english/bound_1